src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/entity/Offence.kt

```diff
@@ -36,4 +36,6 @@ data class Offence(
       if (category == null) return "/" + subCategory.toString().padStart(2, '0')
       return category.toString().padStart(3, '0') + "/" + subCategory.toString().padStart(2, '0')
     }
+  val derivedDescription: String
+    get() = (cjsTitle ?: description)!!
```

Comment:
`!!` is sometimes a code smell - this says that description cannot be null, but the field is nullable and defaults to null... better to make the field non-nullable?

Reply:
`cjsTitle` and `description` come from SDRS.. from their actual data cjsTitle is always populated (so in practice this is not an issue) - however  it's not a mandatory field which is why it's been defined as optional as a belt and braces approach. 

It is a mandatory field as far as NOMIS is concerned (which is what this is used for). I'm thinking maybe we can safely mark `cjsTitle` as mandatory - I'll take it to the team and agree a solution. Good spot